tests/bundle/EventListener/ConditionallyRemoveVaryHeaderListenerTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Tests\Bundle\HttpCache\EventListener;
+
+use EzSystems\PlatformHttpCacheBundle\EventListener\ConditionallyRemoveVaryHeaderListener;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class ConditionallyRemoveVaryHeaderListenerTest extends TestCase
+{
+    /** @var EzSystems\PlatformHttpCacheBundle\EventListener\ConditionallyRemoveVaryHeaderListener */
+    public $conditionallyRemoveVaryHeaderListener;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
```

Comment:
`TestCase::setUp()` is empty and calling it is usually skipped, because it's a placeholder method.
```suggestion
```